ActindoWorkFlow/Operation/03_CompareWorkflows.md

```diff
@@ -0,0 +1,178 @@
+# Compare the workflows versions
+
+The comparison function of the workflow enables to compare different workflow versions. By doing this, you can compare the JSON code lines of two different workflow versions, whereby all differences between the relevant code lines are highlighted.  
+
+You can use this function to do the following: 
+- Compare a specific workflow version to any other version of that workflow available on the current instance.
+- Compare a specific workflow version to any workflow version available on another instance. 
+- Change, insert, or delete code lines. This is possible, because you can edit the JSON code directly.  
```

Comment:
```suggestion
- Change, insert, or delete code lines. This is possible because you can edit the JSON code directly.  
```
Vorschlag: 
Change, insert, or delete code lines, that is, edit the JSON code directly. 
Vielleicht einen Satz als "Warnung" vor den Folgen, i.e. Änderungen in workflow?